compiledb/cli.py

```diff
@@ -47,11 +48,14 @@ def __init__(self, infile, outfile, build_dir, exclude_files, no_build,
 
 
 @click.group(context_settings=CONTEXT_SETTINGS, invoke_without_command=True)
-@click.option('-p', '--parse', 'infile', type=click.File('r'),
+@click.option('-p', '--parse', 'logfile', type=click.File('r'),
               help='Build log file to parse compilation commands from.' +
               '(Default: stdin)', required=False, default=sys.stdin)
+@click.option('-i', '--input', 'infile',
+              help="Input file path (Default: compile_commands.json)",
+              required=False, default='compile_commands.json')
 @click.option('-o', '--output', 'outfile', type=click.File('w'),
-              help="Output file path (Default: std output)",
+              help="Output file path (Default: compile_commands.json)",
```

Comment:
Changed help string since the default value of this arg is 'compile_commands.json`